### ISSUE TITLE
retain_variables=True or addition in loss.backward()

### ISSUE THREAD

**Issue by seann999** [kimhc6028/pytorch-noreward-rl#1]
In [this line](https://github.com/kimhc6028/pytorch-noreward-rl/blob/master/train.py#L159), you use `retain_variables=True` instead of adding the two loss terms for A3C and ICM, which is commented out a few lines later. Is there a difference?

**Comment by kimhc6028** [comment]
There is no difference between them, as far as I know. You could use any of both, though I found none of them were effective in this repo
